skypy/supernovae/__init__.py

```diff
@@ -0,0 +1,4 @@
+"""
+This module contains methods that model the intrinsic properties of supernovae
+Ia and core collapse contamination
```

Comment:
I think the module docstring you proposed is very specific - we are thinking about Ia cosmology and CC contamination now but we are likely to outgrow that scope in the future. Best to keep it general.

```suggestion
This module contains methods that model the intrinsic properties of supernovae.
```

Reply:
That's right. Thanks